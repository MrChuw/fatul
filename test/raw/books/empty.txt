0eNqrVkrKKU0tKMrMK4lPys/PVrKqVsosSc1VskJI6IIldJQSk0syy1LjM/NSUiuUrAx0lMpSi4oz8/OUrIwsDE3MLY3MTc2AyMSithYAVa4dCg==
