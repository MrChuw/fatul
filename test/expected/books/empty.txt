0eNqrVkrKKU0tKMrMK4lPys/PVrKqRogUK1lFx+ooZZak5ipZIYR1wQp1lBKTSzLLUuMz81JSK5SsDHSUylKLijPz85SsjCwMTcwtjcxNzYDIxKK2FgB+cyK0